test/src/api/responder04Test.php

```diff
@@ -0,0 +1,289 @@
perties": [], "sides": 1}, {"description": "Rage Speed Z Swing Die", "skills": ["Rage", "Speed"], "recipe": "Gz(Z)", "value": null, "properties": [], "sides": null}, {"description": "Poison Turbo V Swing Die", "skills": ["Poison", "Turbo"], "recipe": "p(V)!", "value": null, "properties": [], "sides": null}, {"description": "Null Turbo Twin Y Swing Die", "skills": ["Null", "Turbo"], "recipe": "n(Y,Y)!", "value": null, "properties": ["Twin"], "sides": null}, {"description": "Ornery Fire Wildcard die", "skills": ["Ornery", "Fire"], "recipe": "oF(C)", "wildcardPropsArray": {"colour": "black", "type": "Wildcard", "displayedValue": "J<span class=\"suit_black\">&spades;</span>", "suit": "Spades"}, "value": null, "sides": 20, "properties": []}], "swingRequestArray": {"Y": [1, 20], "Z": [4, 30], "V": [6, 12]}, "hasDismissedGame": false, "gameScoreArray": {"L": 0, "W": 0, "D": 0}, "sideScore": null, "canStillWin": null, "turboSizeArray": [], "playerName": "responder003", "prevSwingValueArray": [], "lastActionTime": 1674944556}, {"prevOptValueArray": [], "optRequestArray": [], "playerColor": "#ddffdd", "roundScore": null, "waitingOnAction": false, "playerId": 5, "outOfPlayDieArray": [], "button": {"artFilename": "wildcard.png", "recipe": "(C) (C) (C) (C) (C)", "name": "Wildcard", "originalRecipe": "(C) (C) (C) (C) (C)"}, "capturedDieArray": [], "isChatPrivate": false, "isOnVacation": false, "activeDieArray": [{"description": "Wildcard die", "skills": [], "recipe": "(C)", "wildcardPropsArray": {"colour": "black", "type": "Wildcard", "displayedValue": "Q<span class=\"suit_black\">&clubs;</span>", "suit": "Clubs"}, "value": null, "sides": 20, "properties": []}, {"description": "Wildcard die", "skills": [], "recipe": "(C)", "wildcardPropsArray": {"colour": "black", "type": "Wildcard", "displayedValue": "9<span class=\"suit_black\">&spades;</span>", "suit": "Spades"}, "value": null, "sides": 20, "properties": []}, {"description": "Wildcard die", "skills": [], "recipe": "(C)", "wildcardPropsArray": {"colour": "red", "type": "Wildcard", "displayedValue": "Q<span class=\"suit_red\">&diams;</span>", "suit": "Diamonds"}, "value": null, "sides": 20, "properties": []}, {"description": "Wildcard die", "skills": [], "recipe": "(C)", "wildcardPropsArray": {"colour": "red", "type": "Wildcard", "displayedValue": "A<span class=\"suit_red\">&diams;</span>", "suit": "Diamonds"}, "value": null, "sides": 20, "properties": []}, {"description": "Wildcard die", "skills": [], "recipe": "(C)", "wildcardPropsArray": {"colour": "black", "type": "Wildcard", "displayedValue": "K<span class=\"suit_black\">&clubs;</span>", "suit": "Clubs"}, "value": null, "sides": 20, "properties": []}], "swingRequestArray": [], "hasDismissedGame": false, "gameScoreArray": {"L": 0, "W": 0, "D": 0}, "sideScore": null, "canStillWin": null, "turboSizeArray": [], "playerName": "responder004", "prevSwingValueArray": [], "lastActionTime": 1674944556}], "gameChatLogCount": 0, "validAttackTypeArray": []}', TRUE));
```

Comment:
Oh, sorry, this enormous comment line is my bad.  It's a stray artifact from how i construct a responderTest from replay testing, and can/should be deleted.